modules/local/common/functions.nf

```diff
@@ -1,59 +1,79 @@
-/*
```

Comment:
I don't think the functions.nf is needed anymore for local modules.... see also rnaseq pipeline https://github.com/nf-core/rnaseq/blob/master/modules/local/bedtools_genomecov.nf